survey/survey-application/src/main/java/me/nalab/survey/application/service/findfeedback/type/FeedbackFindByTypeService.java

```diff
@@ -0,0 +1,83 @@
+package me.nalab.survey.application.service.findfeedback.type;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import lombok.RequiredArgsConstructor;
+import me.nalab.survey.application.common.feedback.dto.FeedbackDto;
+import me.nalab.survey.application.common.feedback.mapper.FeedbackDtoMapper;
+import me.nalab.survey.application.common.survey.dto.ChoiceFormQuestionDto;
+import me.nalab.survey.application.common.survey.dto.FormQuestionDtoable;
+import me.nalab.survey.application.common.survey.dto.ShortFormQuestionDto;
+import me.nalab.survey.application.common.survey.dto.SurveyDto;
+import me.nalab.survey.application.common.survey.mapper.SurveyDtoMapper;
+import me.nalab.survey.application.exception.SurveyDoesNotExistException;
+import me.nalab.survey.application.exception.SurveyDoesNotHasTargetException;
+import me.nalab.survey.application.port.in.web.findfeedback.type.FeedbackFindByTypeUseCase;
+import me.nalab.survey.application.port.out.persistence.findfeedback.type.FeedbackFindPort;
+import me.nalab.survey.application.port.out.persistence.findfeedback.type.SurveyFindPort;
+import me.nalab.survey.application.port.out.persistence.findfeedback.type.TargetIdFindPort;
+import me.nalab.survey.domain.feedback.Feedback;
+import me.nalab.survey.domain.survey.Survey;
+
+@Service
+@RequiredArgsConstructor
+public class FeedbackFindByTypeService implements FeedbackFindByTypeUseCase {
+
+	private final SurveyFindPort surveyFindPort;
+	private final FeedbackFindPort feedbackFindPort;
+	private final TargetIdFindPort targetIdFindPort;
+
+	@Transactional
```

Comment:
```suggestion
	@Transactional(readOnly = true)
```

DB에 조회만하고 다른 동작을 하지 않는게, 맞다면 이거 걸어주는게 좋다고 생각합니다 :)